### ISSUE TITLE
osx mojave : safari 12 not working

### ISSUE THREAD

**Issue by chrishough** [beardedspice/beardedspice#837]
Is there a trick to get this to work with the new version of OSX? It detects the tab in safari, but the keyboard shortcuts are not working? Thoughts?

**Comment by KontiKonsumi** [comment]
Also not working with OSX High Sierra Safari 12.

**Comment by chrishough** [comment]
Since I was using this application to only control Amazon Music, I switched to using https://www.keyboardmaestro.com/main/ and the new Amazon Music application.

**Comment by larsenwork** [comment]
@chrishough works fine, so I think this issue can be closed — did you remember to set this https://github.com/beardedspice/beardedspice/wiki/Wont-Work-issue-after-Safari-Update-(9.1.1-and-later)

**Comment by chrishough** [comment]
@larsenwork yes, I have that set.  Since I only use this for Amazon Music right now, I downloaded the AWS application and integrated shortcuts via https://www.keyboardmaestro.com/main/

**Comment by larsenwork** [comment]
@chrishough ah, maybe it's more a Amazon music issue — could you try to see if it works with e.g. youtube?

**Comment by matej-novotny** [comment]
Same for me. Youtube in Safari does not work.
MacOS 10.13.6
Safari 12.0
BeardedSpice 2.2.3

**Comment by macgeekpro** [comment]
@matej-novotny have you turned on "Allow JavaScript from Apple Events" in Develop menu?

**Comment by a-c-m** [comment]
same issue. i get the track data - but it does not control the tab.
